### PR TITLE
feat: update service tags when creating or updating a service

### LINKED ISSUE #383: Tags are not working
When creating a service with tags, the service is created without the specified tags.

Seems the implementation is not completed.

### PR DESCRIPTION
**Pull request by mortenlj**
Fixes #383

### CODE REVIEW

**Comment by byashimov** [comment]
Hey!
Thanks for the contribution. Nice catch.
Could you please restore tags comparison line? Because now it says two maps are equal, but they might be both empty as well.

**Comment by mortenlj** [comment]
> Could you please restore tags comparison line? Because now it says two maps are equal, but they might be both empty as well.

Fair point, fixed now.

**Review by byashimov — APPROVED** [review]
lgtm, thanks for the contribution!

File: tests/postgresql_test.go [hunk 1]
```diff
@@ -106,7 +109,10 @@ func TestPgReadReplica(t *testing.T) {
 	assert.Equal(t, replicaAvn.State, replica.Status.State)
 	assert.Equal(t, replicaAvn.Plan, replica.Spec.Plan)
 	assert.Equal(t, replicaAvn.CloudName, replica.Spec.CloudName)
-	assert.Equal(t, map[string]string{"env": "test", "instance": "replica"}, replica.Spec.Tags)
+	assert.Equal(t, map[string]string{"env": "test", "instance": "foo"}, replica.Spec.Tags)
```
Comment (byashimov) [inline tests/postgresql_test.go:112]: Hmmm. I was too hasty.

Comment (mortenlj) [reply tests/postgresql_test.go:112]: Oh, oops. I did that change with a search-replace, guess I didn't check the diff well enough.

But I think the test passed ... 🤔

**Pull request merged**